core/ratarmountcore/LibArchiveMountSource.py

```diff
@@ -0,0 +1,205 @@
+#!/usr/bin/env python3
+# -*- coding: utf-8 -*-
+
+
+import io
+import os
+import stat
+import time
+from typing import cast, Dict, IO, Iterable, List, Optional, Union
+
```

Comment:
```suggestion

from .compressions import libarchive
```

I'm not 100% sure anymore why but I think this is to be preferred because else `libarchive` is undefined and maybe the style checker didn't like that.

Also, remove the try-import block on line 14-18 when using this.